CloneResponsive/styles.css

```diff
@@ -0,0 +1,812 @@
+}
+
+.content__container__chart__axisX {
+  height: 40px;
+  width: calc(100% - margin-left);
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 20px;
+  margin-left: 50px;
+  margin-bottom: 32px;
+  /* padding-left: 24px; */
+
+  padding-right: 24px;
+  color: #9d9fa2;
+}
+.content__container__chart__marks {
+  height: 40px;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+}
+.content__container__chart__marks,
+.content__container__chart__marks__element {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 22px;
+
+  color: #5f6980;
+}
+.content__container__chart__marks__element {
+  margin-right: 52px;
+}
+
+.content__container__chart__marks__element__green {
+  background-color: #20c997;
+  width: 12px;
+  height: 12px;
+  align-self: center;
+
+  border-radius: 50%;
+  margin-right: 8px;
+}
+.content__container__chart__marks__element__blue {
+  background-color: #5a9cfe;
+  width: 12px;
+  height: 12px;
+  align-self: center;
+  border-radius: 50%;
+  margin-right: 8px;
+}
+.content__container__apps {
+  height: 588px;
+  display: flex;
+  flex-direction: column;
+
+  align-self: stretch;
+  background-color: #fff;
+  border-radius: 12px;
+  margin: 32px 32px 32px 264px;
+}
+.content__container__apps__title {
+  height: 62px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+
+  font-size: 18px;
+  line-height: 30px;
+  color: #282828;
+  padding-left: 24px;
+
+  padding-right: 24px;
+  border-bottom: 1px solid #eaecf0;
+}
+
+.content__container__apps__pagination {
+  height: 92px;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+}
+.content__container__apps__pagination__arrowLeft img,
+.content__container__apps__pagination__arrowRight img {
+  margin-top: 4px;
+  height: 24px;
+}
+.content__container__apps__pagination__number {
+  margin-left: 18px;
+  margin-right: 18px;
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 30px;
+
+  color: #282828;
+}
+
+.content__grid {
+  display: grid;
+  grid-template-columns: repeat(6, auto);
+  grid-template-rows: 54px 76px 76px 76px 76px 76px;
+  justify-items: start;
+  align-items: center;
+  min-width: 850px;
+  overflow-x: hidden;
+}
+.content__grid div {
+  width: 100%;
+  height: 100%;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  border-bottom: 1px solid #eaecf0;
+  color: #5f6980;
+}
+
+.content__container__apps__row__icon {
+  display: flex;
+  flex-direction: row !important;
+  justify-content: flex-start !important;
+  align-items: center;
+  border-bottom: none !important;
+  /* border: 1px solid green; */
+}
+
+.col1__row1 {
+  grid-row: 1/2;
```

Comment:
grid is not necessary here because it is just a table but a big 👍 for grid knowledge